### PR TITLE
Nge 007 add storybook

### PR DESCRIPTION
**Pull request by AdrianRRo**

### CODE REVIEW

File: esbuild.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 const esbuild = require('esbuild');
 const glob = require('tiny-glob');
 (async () => {
-    const tsxPaths = await glob('packages/components/**/*.tsx');
-    const componentPaths = tsxPaths.filter(filePath => !filePath.includes('.spec.tsx'));
-    esbuild.build({
```
Comment (JackBlakeston) [inline esbuild.js:6]: Why don't we filter the tests out anymore?

File: .storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  stories: [
+    '../packages/components/**/*.stories.mdx',
+    '../packages/components/**/*.stories.@(js|jsx|ts|tsx)',
+  ],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
```
Comment (JackBlakeston) [inline .storybook/main.js:9]: No need to add @storybook/preact here??

File: packages/components/molecules/Counter/Counter.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  parameters: {
+    // Specify that the component is a Web Component
+    webComponents: {
+      // Set the tag name of the Web Component
+      tagName: 'my-counter',
+    },
+  },
+};
+
+export const DefaultMolecules = () => <my-counter />;
```
Comment (JackBlakeston) [inline packages/components/molecules/Counter/Counter.stories.tsx:15]: Do you want to name these differently according to the atomic type of the component?

**Review by JackBlakeston — APPROVED** [review]
All good just a few questions that I have

File: esbuild.js (hunk 1 above)
Comment (AdrianRRo) [reply esbuild.js:6]: now we transpile the register.ts files, however I have changed it to make it clearer.

**Pull request merged**